tests/foreman/ui/test_computeresource_libvirt.py

```diff
@@ -386,3 +386,123 @@ def test_positive_associate_with_custom_profile_with_template():
+            'cpus': 1,
+            'memory': 6442450944,
+            'firmware': 'Automatic',
+            'volumes_attributes': {'0': {'capacity': '10G', 'format_type': 'qcow2'}},
+            'nics_attributes': {
+                '0': {
+                    'type': 'bridge',
+                    'bridge': f'br-{settings.provisioning.vlan_id}',
+                },
+            },
+        },
+    ).create()
+
+    # Add compute-profile and compute-resource to hostgroup
+    sat.cli.HostGroup.update(
+        {
+            'id': provisioning_hostgroup.id,
+            'compute-profile': COMPUTE_PROFILE_SMALL,
+            'compute-resource-id': cr_info.id,
+        }
+    )
+
+    request.addfinalizer(lambda: sat.provisioning_cleanup(host_fqdn))
+
+    @request.addfinalizer
+    def _finalize():
+        cr = sat.api.LibvirtComputeResource().search(query={'search': f'name={cr_info.id}'})
+        if cr:
+            sat.api.LibvirtComputeResource(id=cr[0].id).delete()
+
+    # Create host
+    with session:
+        session.organization.select(org_name=module_sca_manifest_org.name)
+        session.location.select(loc_name=module_location.name)
+        session.host.create(
+            {
+                'host.name': hostname,
+                'host.organization': module_sca_manifest_org.name,
+                'host.location': module_location.name,
+                'host.hostgroup': provisioning_hostgroup.name,
+                'host.inherit_deploy_option': False,
+                'provider_content.operating_system.provision_method': True,
+            }
+        )
+
+        # Check on Libvirt, if VM exists
+        result = sat.execute(
+            f'su foreman -s /bin/bash -c "virsh -c {LIBVIRT_URL} list --state-running"'
+        )
+        assert hostname in result.stdout
+
+        # check the build status
+        host_fqdn = f'{hostname}.{provisioning_hostgroup.domain.read().name}'
+        wait_for(
+            lambda: session.host_new.get_host_statuses(host_fqdn)['Build']['Status']
+            != 'Pending installation',
+            timeout=1800,
+            delay=30,
+            fail_func=session.browser.refresh,
+            silent_failure=True,
+            handle_exception=True,
+        )
+        values = session.host_new.get_host_statuses(host_fqdn)
+        assert values['Build']['Status'] == 'Installed'
+
+        # Verify SecureBoot is enabled on host after provisioning is completed successfully
+        if pxe_loader.vm_firmware == 'uefi_secure_boot':
+            host_info = sat.api.Host().search(query={'search': f'name="{host_fqdn}"'})
+            for host in host_info:
+                provisioning_host = ContentHost(host.ip, auth=module_ssh_key_file)
+                # Wait for the host to be rebooted and SSH daemon to be started.
+                provisioning_host.wait_for_connection()
+                assert (
+                    'SecureBoot enabled' in provisioning_host.execute('mokutil --sb-state').stdout
```

Comment:
I wondered how this worked for you in PRT, since SSH is disabled by default for EL9+ hosts, and then I discovered you're not using or passing Firmware/PXE loader during host creation, so I guess this condition is getting skipped for Secureboot